apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/[resourceType]/[resource]/index.vue

```diff
@@ -506,8 +515,15 @@ const organisations = computed(() => resource.value.organisationsInvolved);
     <template #header>
       <PageHeader
         id="resource-page-header"
-        :title="resource?.acronym || resource.name"
-        :description="resource?.acronym ? resource.name : ''"
+        :title="
+          route.params.resourceType === 'about'
+            ? 'About '
+            : resource?.acronym || resource.name
```

Comment:
How does the binding work here? is it 
`(route.params.resourceType === 'about' ? 'About ' : resource?.acronym) || resource.name`
or:
`route.params.resourceType === 'about' ? 'About ' : (resource?.acronym || resource.name)`

would be nice to make this explicit (add brackets) to avoid confusion.

Reply:
your codes are equivalent.

however, you might be more satisfied by the second?

Reply:
they are not the same, the first one is a bug, the second is correct. 
For the first one, the `|| name` part will never trigger. Combining logic and ternaries is risky and I'm always unsure of the binding strength, so being explicit with brackets is I think necessary in those cases.

Reply:
you are right, changed it